### ISSUE TITLE
PORT is no respond from live web server

### ISSUE THREAD

**Issue by sathishsanthoshsusidhar** [Xowap/PHP-Serial#29]
Hai, I using php Serial library in local xampp server its working fine. But i moved same code to live its not working.  How can i solve this issue?.

**Comment by denmind** [comment]
Is your path to the PhpSerial file correct?

**Comment by sathishsanthoshsusidhar** [comment]
Yes, But my goal is can access my local system Serial port like COM3.

**Comment by denmind** [comment]
We might be working on something similar, and I have been getting difficulties in finding a solution. So what I did is, I just abandoned the concept of any php serial port communication, probably because : a)most of then are deprecated )and they are heavily based for LINUX systems.
